server/src/test/java/org/opensearch/cluster/service/MasterServiceTests.java

```diff
@@ -97,16 +98,15 @@
 public class MasterServiceTests extends OpenSearchTestCase {
 
     private static ThreadPool threadPool;
-    private static long relativeTimeInMillis;
+    private static Long relativeTimeInMillis;
+
+    private static LongSupplier nanoTimeSupplier() {
+        return () -> relativeTimeInMillis * 1000000L;
```

Comment:
Would prefer a constant here rather than a magic number, e.g. NANOS_PER_MILLIS.  I'm also fond of using underscores to make it easy to count zeroes.  Less important as this is a test class.

Reply:
Thanks, I guess we might be moving away with this change to have a method in ThreadPool, but I'll account this if applicable in further changes.